README.md

```diff
@@ -1,6 +1,32 @@
 # AirPyllution
+A package for visualizing future, historic and current air pollution data using the [OpenWeather API](https://openweathermap.org).
```

Comment:
Is it only about visualization ? Since we have a function or two which doesn't return a graph.